scripts/state/mega-migrate.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
```

Comment:
should this file be called as a part of a build process?

Reply:
I don't think we have a script to run it. We can add to CI later @ailisp